tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py

```diff
@@ -190,68 +156,41 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
           of the monitored file when the old content is lower than the allowed limit or
           the testing platform is Windows.
 
-    input_description: A test case (ossec_conf_report) is contained in external YAML file (wazuh_conf.yaml)
-                       which includes configuration settings for the 'wazuh-syscheckd' daemon and, these
-                       are combined with the testing directory and files to be monitored defined in the module.
+    input_description: The file 'configuration_large_changes.yaml' provides the configuration template.
+                       The file 'cases_large_changes.yaml' provides the tes cases configuration
```

Comment:
```suggestion
                       The file 'cases_large_changes.yaml' provides the test cases configuration
```

Reply:
Done in 86af83d46813890fa2635bd6053d9dc80376c96c